UKHomeOffice/cerberus-service
# Cop 9273 - Added field key (field name) highliting for changes between task versions

## Description
This PR adds field key highliting that shows which fields has changed. This change also makes it easier for the 
user to still tell which field has changed, even when it has no value.

## To Test
- Pull and run against dev.
- Navigate to the tab titled "**In progress**".
- Locate "**JONNY-2021-11-16-1 Pre-Arrival**" and open, this should have 5 versions.
- In each version, if there is any change, the field name will be highlighted along with the field's value. If there is a change and it is an empty string, the field value should still be highlited to reflect the change.


## Developer Checklist

\* Required
